docs/edit-contracts.md

```diff
@@ -0,0 +1,193 @@
+# How to edit zkEVM contracts for Kurtosis CDK
+
+A step-by-step guide to edit the zkEVM Solidity contracts in a Polygon CDK devnet.
+
+This document draws from the following resources:
+
+- [Polygon CDK Kurtosis Package](../README.md)
+- [Custom Docker Images for Kurtosis CDK](../docker/README.md)
+- [Video guide to the Polygon CDK](https://www.youtube.com/watch?v=6ykNLEhwxIs)
+
+## Goals
+
+- Edit the Solidity code in repo `zkevm-contracts` in file `FflonkVerifier.sol` in function `verifyProof`.
+- Rebuild the `zkevm-contracts` docker image.
+- Edit this repo `kurtosis-cdk` to use our new build of `zkevm-contracts`.
+- Spin up a Polygon CDK devnet and observe our changes in the active devnet.
+
+# Set up your system
+
+Follow the instructions at [Polygon CDK Kurtosis Package](../README.md) up to the terminal commands
+
+```bash
+kurtosis clean --all
+kurtosis run --enclave cdk-v1 --args-file params.yml --image-download always .
+```
+
+These commands will spin up a Polygon CDK devnet on your local machine. Feel free to follow subsequent instructions in that document to play with your devnet.
+
+Let's take a look at the Solidity code we want to change. Attach a shell to `contracts-001` container:
+
+```bash
+kurtosis service shell cdk-v1 contracts-001
+```
+
+In the attached shell take a peek at the Solidity code for `verifyProof`:
+
+```bash
+grep -A 1 "verifyProoof" zkevm-contracts/contracts/verifiers/FflonkVerifier.sol
```

Comment:
```suggestion
grep -A 1 "verifyProof" zkevm-contracts/contracts/verifiers/FflonkVerifier.sol
```

Reply:
fixed in https://github.com/EspressoSystems/kurtosis-cdk/pull/2/commits/974e2dc898135000ad8eb66c13764644077576e5